birthdays-project/BirthdayApp/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+
+class Birthdate(models.Model):
+    dob = models.DateField(max_length=8)
+
+    # Displays date in a dd-MMM-yy format.
+    def __str__(self):
+        return '{:%d %b %y}'.format(self.dob)
+
+
+# Creates a user Model.
+class User(models.Model):
```

Comment:
I did have some difficulty using User, and it was probably due to the fact that Django has a User table. In our theoretical situation, I imagined this being an office app so I decided to use Employee as I think it fits better and makes for a better url.  I imagined domain.com/employee/1 instead of domain.com/person/1.  I looked back on the original scenario and you didn't specify that it was an office situation, so I could use Person if you think that's best.